include/deal.II/grid/grid_tools.h

```diff
@@ -2696,24 +2696,32 @@ namespace GridTools
   /**
    * In this collective operation each process provides a vector
    * of bounding boxes and a communicator.
-   * All these vectors are gathered in each process
-   * and organized in a search tree which is then returned.
-   *
-   * The idea is that the vector of bounding boxes describes
-   * a relevant property which could be of use to other processes,
-   * e.g. for a distributed triangulation, the bounding
-   * boxes could describe the portion of the mesh which is
-   * locally owned by the current process.
-   *
-   * The search tree is an r-tree with packing algorithm,
-   * which is provided by boost library.
+   * All these vectors are gathered on each of the processes,
+   * organized in a search tree which, and then returned.
+   *
+   * The idea is that the vector of bounding boxes describes a
+   * relevant property of the computations on each process
+   * individually, but which could be of use to other processes. An
```

Comment:
Somehow this clause sounds strange to me. Maybe
```suggestion
   * individually, which could also be of use to other processes. An
```
?